src/Producers/Builder.php

```diff
@@ -158,6 +160,14 @@ public function withDebugDisabled(): self
         return $this->withDebugEnabled(false);
     }
 
+    public function withFlushOptions(int $retries, int $timeoutInMs): self
```

Comment:
I think one method for each config option is better than having one for both. If someone wants to change the number of retries, they would also have to set the timeout as it is a required parameter for this method

Reply:
I agree, that might be cleaner! I will try and update it next week when I find some time :)